notebooks/python/python-basics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>Consider saying "There is a built-in Python function called <code>round</code> that we can use to round..." rather than "We can use the <code>round</code> function."</p><p> </p><p>Also wondering if a very brief mention that there are both "built-in" and user-defined functions in python?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ESCIP/escip.github.io/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>